modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -71,6 +72,22 @@ type HTTPTransport struct {
 	// connecting to an upstream. Default: off.
 	ProxyProtocol string `json:"proxy_protocol,omitempty"`
 
+	// URL to the server that the HTTP transport will use to proxy
+	// requests to the upstream. See http.Transport.Proxy for
+	// information regarding supported protocols.
+	//
+	// Providing a value to this parameter results in
+	// requests flowing through the reverse_proxy in the following
+	// way:
+	//
+	// User Agent ->
+	//  reverse_proxy ->
+	//  proxy_protocol ->
+	//  transport_proxy_url -> upstream
+	//
+	// Default: off.
+	TransportProxyURL string `json:"transport_proxy_url,omitempty"`
```

Comment:
I think `transport` within the "Transport" module is redundant. We could either call it "forward proxy", or just "proxy" (but the latter might be confusing because we're in a "reverse proxy" already).

```suggestion
	ForwardProxyURL string `json:"forward_proxy_url,omitempty"`
```

Reply:
That's a really good point!